drivers/dma/Kconfig.mcux_edma

```diff
@@ -33,4 +49,18 @@ config DMA_MCUX_USE_DTCM_FOR_DMA_DESCRIPTORS
 	  When this option is activated, the descriptors for DMA transfer are
 	  located in the DTCM (Data Tightly Coupled Memory).
 
+config DMA_MCUX_HAS_ERROR_IRQ
```

Comment:
put to soc kconfig, not here

Reply:
this Kconfig is indicating if the error interrupt is implemented, and is part of the DMA hw IP. So it should be in this Kconfig file. The SoC should select this or not depending on the implementation.

Reply:
if in this case, put them in dts would be better.

Reply:
okay, i've pushed to update